### PR TITLE
AM feature navigation

### PR DESCRIPTION
**Pull request by amarquesmoura**
this changes the menu style from tabBar to drawer

### CODE REVIEW

**Review by teelau — DISMISSED** [review]

**Comment by teelau** [comment]
There will be gitconflicts with the .gitignore, 
https://github.com/redacademy/bikelocker-spring-2019/pull/50

Jeremiah is also merging a dependency for pods.

**Comment by siddharthparmar7** [comment]
Please resolve merge conflicts.

**Comment by amarquesmoura** [comment]
Done.

File: client/js/navigation/NavigationLayout.js [hunk 1]
```diff
@@ -56,41 +65,49 @@ const ProfileStack = createStackNavigator(
...
+    Resources: {
+      screen: ResourcesStack,
+      navigationOptions: ({ navigation }) => ({
+        title: "Helpful Resources"
+      })
+    }
+  },
+  {
+    defaultNavigationOptions: ({ navigation }) => ({}),
+    drawerWidth: 250,
```
Comment (teelau) [inline client/js/navigation/NavigationLayout.js:100]: Can this be a percentage of the width instead of pixel values?

**Review by siddharthparmar7 — APPROVED** [review]
Nice work!

**Pull request merged**